### PR TITLE
DOC: Fixing EX01 - Added examples

### PR DESCRIPTION
**Pull request by DeaMariaLeon**
- [x] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).

Towards https://github.com/pandas-dev/pandas/issues/37875

### CODE REVIEW

**Comment by mroeschke** [comment]
```
Error: /home/runner/work/pandas/pandas/pandas/core/resample.py:1106:RT01:pandas.core.resample.Resampler.max:No Returns section found
Error: /home/runner/work/pandas/pandas/pandas/core/resample.py:1076:RT01:pandas.core.resample.Resampler.min:No Returns section found
```

File: pandas/core/groupby/generic.py [hunk 1]
```diff
@@ -646,6 +647,21 @@ def nunique(self, dropna: bool = True) -> Series | DataFrame:
...
+
+        >>> ser = pd.Series([1, 2, 3, 3], index=pd.DatetimeIndex(
+        ...                 ['2023-01-01', '2023-01-15', '2023-02-01', '2023-02-15']))
+        >>> ser
+        2023-01-01    1
+        2023-01-15    2
+        2023-02-01    3
+        2023-02-15    3
+        dtype: int64
+        >>> ser.resample('M').nunique()
```
Comment (MarcoGorelli) [inline pandas/core/groupby/generic.py:661]: the examples look good!

any chance we could make an example which uses 'D' here instead please, so as to not add even more files to update in #52064?

Comment (DeaMariaLeon) [reply pandas/core/groupby/generic.py]: Changed all to `MS` to avoid too many changes, as per our conversation.

**Review by MarcoGorelli — APPROVED** [review]
thanks @DeaMariaLeon !

**Pull request merged**